packages/hash/api/src/auth/seed-dev-users.ts

```diff
@@ -17,12 +16,12 @@ const devUsers: readonly DevelopmentUser[] = [
   {
     email: "alice@example.com",
     shortname: "alice",
-    preferredName: "Alice Alison",
+    preferredName: "Alice",
   },
   {
     email: "bob@example.com",
-    shortname: "bob",
-    preferredName: "Bob Bobson",
+    shortname: "bob01",
```

Comment:
FYI @thehabbos007 our previous validation logic was that shortnames have to be longer than 4 characters - perhaps this is something we revisit, but for now have modified this accordingly.

Reply:
Sure, done in https://github.com/hashintel/hash/pull/950/commits/4c5573ffcb02f1929ce25196b162e3a01010c18e